src/sign-in-xplat/README.md

```diff
@@ -0,0 +1,113 @@
+1. [Visual Studio 2022 Preview](https://visualstudio.microsoft.com/vs/preview/#download-preview)
+1. [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+1. [.NET MAUI](https://docs.microsoft.com/en-us/dotnet/maui/get-started/installation)
+
+## Setup
+
+### 1. Register the web API application in your Azure Active Directory
+
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the sample app.
+
+Use the following settings for your app registration:
+
+| App registration <br/> setting | Value for this sample app                                         | Notes                                                                                                       |
+|:------------------------------:|:------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
+| **Name**                       | `active-directory-dotnet-xplat-maui`                              | Suggested value for this sample. <br/> You can change the app name at any time.                             |
+| **Supported account types**    | **My organization only**                                          | Required for this sample. <br/> Support for the Single tenant.                                              |
+| **Platform type**              | `Mobile and desktop applications`                                 | Required value for this sample. <br/> Enables the required and optional settings for the app type.          |
+| **Redirect URIs**              | `https://login.microsoftonline.com/common/oauth2/nativeclient`    | Required value for this sample. <br/> You can change that later in your own implementation.                 |
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box or select in the Azure portal.
+
+<details>
+   <summary>:computer: Alternative: Register the application using az-cli</summary>
+
+1. Register a new Azure AD App with a reply url
+
+   ```bash
+   AZURE_AD_APP_CLIENT_ID_XPLAT=$(az ad app create --display-name "active-directory-dotnet-xplat-maui" --reply-urls "https://login.microsoftonline.com/common/oauth2/nativeclient" --native-app true --query appId -o tsv)
+   ```
+
+</details>
+
+### 2. Configure the web app
+
+1. Open the `XPlat.csrpoj` under the the `sign-in-xplat` folder in your code editor.
+1. Open the `MainPage.xaml.cs` file and modify the following code:
+
+    ```csharp
+    private const string ClientId = "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]";
```

Comment:
@ckittel We need a formatting convention for these replaceable token values - they're sorta all over the place in these samples. Then we need a pass through them all to adhere to the convention.

Unless the following goes against the language convention, I propose:

- short
- descriptive
- all CAPS
- NO_SPACES - separate words with underscore (`_`)

```csharp
// NOT our style:
private const string ClientId = "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]";

// Our style:
private const string _clientID = "APPLICATION_(CLIENT)_ID";
```

Reply:
I could totally see that working well.

This would be another great task for the content development team members who want to be involved in the creation of the code artifacts, like we heard.  Our team could pause and do this as well, but personally, I'd like to see our team continue to focus on the bulk of the scenario coverage and then leave any unification or presentational tweaks to those writing the presentation layer for this all.  Open to any path though here.

Reply:
> I'd like to see our team continue to focus on the bulk of the scenario coverage and then leave any unification or presentational tweaks to those writing the presentation layer for this all.

I'm cool with that for the retroactive updates. Are you open to using that format for new samples, though?

Reply:
Indeed!

Reply:
thanks for the feedback @mmacy, for future changes on this we're looking forward to the retroactive updates. 

done | addressed from [3993d](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/15/commits/3993d0cfda18a81c78cc8edc3ba5dc4b604d127b)